Python-Tutorial_01.ipynb

```diff
@@ -228,7 +416,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.5"
+   "version": "3.7.1"
```

Comment:
nit: If you bump the version here should you not bump the version of each / all the notebooks.